configs/setups/awicm3/awicm3.yaml

```diff
@@ -1,196 +1,657 @@
         namelist_changes:
-                namelist.jsbach:
-                        hydrology_ctl:
-                                gethd: "remove_from_namelist"
-                                puthd: "remove_from_namelist"
-        version: "3.20"
-
-        choose_computer.name:
-                ollie:
-                        dynveg_file_ending: ""
-                        no_dynveg_file_ending: ""
-
-
-#########################################################################################
-
-
-fesom2:
-        choose_general.version:
-                2.0:
-                        version: "2.0"
+                fort.4:
+                        NAERAD:
+                                CMIP5DATADIR: "${cmip5_data_dir_nml}"
+                                CRTABLEDIR: "${rtables_dir}"
+                                LECOMPGRID: "${lecompgrid}"
+                        NAMCLDP:
+                                SCLCT_SWITCH: "${sclct_switch}"
+                        NAMFPG:
+                                NFPMAX: "${res_number}"
+                        NAMFPD:
+                                NLAT: "${nlat}"
+                                NLON: "${nlon}"
+                        NAMPAR0:
+                                NSPECRESMIN: "${nspecresmin}"
+                        NAMDYNA:
+                                LGRADSP: "${lgradsp}"
+                        NAMNMI:
+                                LASSI: "${lassi}"
+                        NAMCT0:
+                                LFDBOP: "${lfdbop}"
+                                LSMSSIG: "${lsmssig}"
+                        # Where the hell do these namelist changes come from? I don't even remember writting them down... and they are not part of the original fort.4???
+                        #NAMGRIB:
+                        #        NJTKDATE: "${start_date}"
+                        #        NJTKOFFSETSTEP: "${steps_since_initial}"
+                        NAMORB:
+                                LCORBMD:  "${orb_switch}"
+                                ORBMODE: "${orb_mode}"
+                                ORBIY: "${orb_iyear}"
+        # Namelist changes for wam_namelist not included yet as the first test is for DRH6
+        # that has no wave model. TODO: implement in the near future
+
+        runtime_environment_changes:
+                add_export_vars:
+                        # Turn on FESOM coupling
+                        - 'FESOM_USE_CPLNG="active"'
+                        - 'ECE_CPL_NEMO_LIM="false"'
+                        - 'ECE_CPL_FESOM_FESIM="true"'
+                        - 'ECE_AWI_CPL_FESOM="true"'
+
+        res_number_tl: ${resolution}${levels} # @JOAKIM: what is this variable for? @JAN: Is the directory correct?
+        choose_general.resolution:
+                TL159_CORE2:
+                        nproca: 279 # @JAN, are these numbers correct for this resolution?
```

Comment:
Yes, as long as we are not using tasksets and MPI/OMP. Once we use MPI/OMP, we need full nodes for oifs. But for now this is correct!

Reply:
OpenIFS will not accept prime numbers as nproca (don't ask me why...). 
So any multiple of 2 works, or 279 = 31 * 9 works too. 
I often use 287 = 41 * 7, + 1 for rnfmap = 288 which fits well on 36, 48 or 96-core nodes.

Reply:
Yeah, perhaps a comment in this line with a nod to the prime problems would be appropriate. Once you do MPI/OMP your nproca*OMP_NUM_THREADS has to fill a node. Then the rules are different again.